lua/gatekeeper_config.lua

```diff
@@ -26,9 +26,14 @@ function gatekeeper_init()
 
 		local gguf = require("ggu")
 		local ggu_conf = gguf(net_conf, gk_conf, ggu_lcore)
+
+		local dyf = require("dyn_cfg")
+		local dy_conf = dyf(numa_table)
 	else
 		local gtf = require("gt")
 		local gt_conf = gtf(net_conf, numa_table)
+
+		-- TODO Add Dynamic Configuration to GT blocks.
```

Comment:
Why don't you move the dynamic block initialization above after this `if` statement, so both Gatekeeper and Grantor servers have it? It would enable you to drop this `TODO`.